Resources/Prototypes/_NF/Entities/Clothing/Neck/cloaks.yml

```
@@ -16,4 +16,13 @@
   description: A top of the line cloak for Frontier Outpost's station representative. Made from exquisite fibers and furs, this thing must have cost a fortune! The fact this cloak was even issued shows the trust Central Command is placing in their representative on this outpost.
   components:
   - type: Sprite
-    sprite: _NF/Clothing/Neck/Cloaks/sr.rsi
\ No newline at end of file
+    sprite: _NF/Clothing/Neck/Cloaks/sr.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckCloakPirateParrot
+  name: polly the pirate parrot
+  description: A mute parrot named polly, that enjoys crackers and cannons.
```

Proper nouns.  Also, could give a tip that it's actually equippable.
```suggestion
  name: Polly the pirate parrot
  description: A mute parrot named Polly. Loves sitting on shoulders, crackers, and cannons.
```